testing/build/gen_fixtures_location_symbol.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# Copyright 2017 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import argparse
+import subprocess
+import sys
+import os
+
+
+def main():
+  parser = argparse.ArgumentParser(
+      description='Create the symbol specifying the location of test fixtures.')
+
+  parser.add_argument('--fixtures_location_file', type=str, required=True)
+  parser.add_argument('--fixtures_location', type=str, required=True)
+
+  args = parser.parse_args()
+
+  with open(args.fixtures_location_file, 'w') as file:
+    file.write("namespace testing {const char* GetFixturesPath() {return \"%s\";}}"
```

Comment:
use single quotes around this string for consistency (and to avoid escaping the inner double quote)